test/test-hash.4th

```diff
@@ -0,0 +1,15 @@
+\ Copyright 2019 nomennescio
+s" string hash" describe#{
+  s" short strings" it#{
+    <{ s" Hello World!" s# -> s" Hello World!" s# }>
+    <{ s" Hello Worlds!" s# -> s" Hello Worlds!" s# }>
+  }#
+  s" failing compares" it#{
+    <{ s" Hello World!" s# -> s" Hello Worlds!" s# }>
+    <{ s" Hello Worlds!" s# -> s" Hello Worlds! " s# }>
+  }#
+  s" failing compares with custom messages" it#{
+    <{ s" Hello World!" 2dup ." Actual: '" type ." '" s# -> s" Hello Worlds!" 2dup ." , Expected: '" type ." '" s# }>
```

Comment:
Can you add passing test with messages?

Reply:
Sure, but this 'commit' is getting heavier and heavier, and my time is running out. I propose to leave it as it now is, and postpone any further improvements on custom reporting.

Reply:
You added what I wanted to see in https://github.com/Codewars/ttester-codewars/pull/4/commits/61c244fc67bb7cbf816dd09c2745091ad7972275 so I'm ok to merge this.